tensorflow/core/common_runtime/gpu/gpu_process_state.cc

```diff
@@ -53,6 +54,12 @@ bool useCudaMemoryGuardAllocator() {
          std::strcmp(debug_allocator_str, "memory_guard") == 0;
 }
 
+bool useCudaMallocAsyncAllocator() {
```

Comment:
style: function names should start with a capital letter.

Reply:
They're local to the file and have a single call-site. While you're here, can you just change them to have the correct style?

Reply:
done.